coderedcms/models/page_models.py

```diff
@@ -827,7 +827,7 @@ def most_recent_occurrence(self):
 
     @property
     def seo_struct_event_dict(self) -> dict:
-        next_occ = self.most_recent_occurrence
+        next_occ = self.most_recent_occurrence[2]
```

Comment:
Can you add a comment as to what these magic number indexes in the tuple represent (perhaps even a type hint)?